Documentation/contributing.rst

```diff
@@ -1130,6 +1130,22 @@ Logging into VM running tests
     $ JOB_BASE_NAME=PR-1588 BUILD_NUMBER=6 vagrant ssh 6e68c6c
 
 
+Jenkinsfiles extensions
+------------------------
+
+In cilium a `Jenkins Library <https://github.com/cilium/Jenkins-library>`_ is in
+use to gather metadata from PRs and to improve our Jenkinsfiles, the exported
+methods are:
```

Comment:
"Cilium uses a custom `Jenkins helper library <https://github.com/cilium/Jenkins-library>`_  to gather metadata from PRs and simplify our Jenkinsfiles. The exported methods are:" reads a little easier for me. Without the link target it wasn't clear to me which library it was, and even then it took some time for me to realise it was custom.